extensions/amp-image-slider/amp-image-slider.md

```diff
@@ -110,6 +110,8 @@ reappear if it then leaves the viewport and goes back in again. To stop such
 reappearing behavior, add the `disable-hint-reappear` attribute to the image
 slider.
 
+> Important: This attribute is renamed as `display-hint-once` in Bento Version `1.0`.
+
```

Comment:
We should have separate markdown files for the two versions. Please:
- copy this file into the 0.1 directory unchanged
- modify the root file (which going forward will be strictly about 1.0) with the following:
  - include `bento: true` and `experimental: true` in the top frontmatter annotation
  - add a bottom section `## Version Notes` which discusses the attribute rename